t/http-reverse-proxy.t

```diff
@@ -201,6 +205,28 @@ END {
     );
 }
 
+# Modifying request's body
+{
+    my $proxy-app = Cro::HTTP::ReverseProxy.new(
+        to => "http://localhost:{HTTP_TEST_PORT_A}/",
+        request => sub ($request) {
+            $request.set-body("body");
+        },
+        response => sub ($response) {
+            is $response.body-text.result, 'body', 'Proxy set a new body in request';
+            $response.set-body('body updated');
+        });
+
+    my $proxy = Cro::HTTP::Server.new(port => HTTP_TEST_PORT_PROXY, application => $proxy-app);
+    $proxy.start;
+    LEAVE $proxy.stop;
+
+    my $c = Cro::HTTP::Client.new(base-uri => "http://localhost:{HTTP_TEST_PORT_PROXY}");
+    given await $c.get('/base-body') -> $resp {
```

Comment:
Should probably be a `POST` or `PUT` request if it's going to have a request body.